pkg/go/.golangci.yaml

```diff
@@ -2,75 +2,75 @@ run:
   timeout: 3m
```

Comment:
This change was suggested by me, aligning the linter with that of OpenFGA since these modules will now be sourced in the core repo and contributions will be made to from the core team so that there is a more similar flow to the core repo.